run/helm/snap/templates/config/_task.tpl

```diff
@@ -0,0 +1,19 @@
+---
+max-failures: 10
```

Comment:
I totally agree that these can go in their own template files and not in `values.yaml`. I was just wondering why they were complete files. If there isn't anything that needs to be reused as a golang template, then why not just put it in a `ConfigMap` in a file called `config.yaml` or something similar? Does that make any more sense?